docs/mvvm/source-gen/diagnostics/MVVMTK0013.md

```diff
@@ -0,0 +1,51 @@
+---
+title: MVVM Toolkit error MVVMTK0013
+author: Sergio0694
+description: MVVM Toolkit error MVVMTK0013
+keywords: community toolkit, dotnet, csharp, mvvm, net core, net standard, source generators
+dev_langs:
+  - csharp
+---
+
+# MVVM Toolkit error MVVMTK0013
+
+Cannot apply the `[RelayCommand]` attribute specifying to include a cancel command to methods not mapping to an asynchronous command type accepting a cancellation token.
+
+The following sample generates MVVMTK0013:
+
+```csharp
+using CommunityToolkit.Mvvm.Input;
+
+namespace MyApp;
+
+public partial class SampleViewModel
+{
+    // The target method is synchronous, so no cancel command can be generated
+    [RelayCommand(IncludeCancelCommand = true)]
+    private void GreetUser(User user)
+    {
+    }
+}
+```
+
+The same applies to an asynchronous method that does not take an input token:
```

Comment:
It'll help when we have proper docs, but I do find with a lot of these, seeing the wrong example doens't help me get to the right solution. Like I need to know to add an argument of `CancellationToken token` right?

I know these are patterned off the C# API error messages, but I find it frustrating sometimes there too. Like sometimes seeing the simple error case is helpful. But if I'm here, I've already got the error. Sometimes I just need to understand what I'm supposed to be doing instead? (as I clearly didn't understand (or look for) the original docs to get to this point).

Reply:
Yeah agreed, we might want to expand these in a follow up PR suggesting how to fix it.
I'd say we should do that separately though, and decide a good pattern to make that consistent.
I do agree it'd probably help folks, for sure 😊